GVFS/GVFS.Common/Prefetch/Git/DiffHelper.cs

```diff
@@ -312,7 +315,12 @@ private void EnqueueOperationsFromDiffTreeLine(ITracer activity, string repoRoot
         }
 
         private bool ShouldIncludeResult(DiffTreeResult blobAdd)
-        {
+        {
+            if (blobAdd.TargetIsSymlink && !this.includeSymlinks)

```

Comment:
I'm not happy with this as it breaks the pattern of the rest of this function, I'm open to suggestions on how revisit this.

Reply:
How is it breaking the pattern?